caffe2/python/ideep/conv_op_test.py

```diff
@@ -158,152 +158,6 @@ def test_depthwise_convolution(self, batch_size, gc, dc):
             print(np.max(np.abs(Y2 - Y0)))
             self.assertTrue(False)
 
-    @unittest.skipIf(sys.version_info.major > 2, "broken in python 3")
```

Comment:
Probably shouldn't delete this, just mark it as expected broken or something? Or maybe the code is moribund and we don't care anyway... I guess probably that is true.